src/css/quasar.variables.sass

```diff
@@ -3,3 +3,4 @@ $secondary : #EF7D18
 $tertiary  : #FFF3E9
 $accent    : #fef6ef
 $background: #FFFFFF
+$inputcolor: rgb(247, 247, 247)
```

Comment:
I am okay with rgb values, but just for consistency maybe Hex-value is better ;)